testpilot-metrics.js

```diff
@@ -95,18 +96,25 @@ Metrics.prototype = {
    * should return an object whose keys are GA Measurement Protocol parameters.
    * The returned object will be form encoded and sent to GA.
    */
-  sendEvent: function({method, object=null, category='interactions', variant=null} = {}, transform) {
+  sendEvent: function({method, object, category, variant} = {}, transform) {
+    const args = {
+      method: method,
+      object: object || null,
+      category: category || 'interactions',
+      variant: variant || null
+    };
```

Comment:
Ah, this is wrong. I need to copy over all the keys, then insert defaults. Fixing now